.github/workflows/ci_docs.yml

```diff
@@ -6,12 +6,14 @@ on:
       - 'lib/**.nim'
       - 'doc/**.rst'
       - 'doc/nimdoc.css'
+      - '.github/workflows/ci_docs.yml'
 
   pull_request:
     # Run only on changes on these files
     paths:
       - 'lib/**.nim'
       - 'doc/**.rst'
```

Comment:
`'doc/nimdoc.css'` missing, is that intentional ? if so maybe add a comment explaining

Reply:
I thought that'd be obvious. Changes to the css file won't change CI result (unless you delete it, but it's unlikely that'd even pass review). We rebuild it on push only to update the css in the published docs.